runtime/doc/vim_diff.txt

```diff
@@ -38,7 +39,13 @@ these differences.
 - 'wildmode' defaults to "list:longest,full"
 
 ==============================================================================
-3. Changed features					 *nvim-features-changed*
+3. New keymaps        			   		          *nvim-keymaps*
+
+A small set of keymaps has been changed in nvim's default configuration. These
+are not hardcoded, and can be disabled by the user (see |nvim-extra-keymaps|).
+
+==============================================================================
+4. Changed features					 *nvim-features-changed*
```

Comment:
I think after this PR it might be a good idea to switch all new documentation to only spaces (I could open an issue). 

IMHO, tabs make sense for code where people might use different tabstop values, but not for documentation such as this. We already have `ts=8` in all modelines for instances of `doc/*.txt`. Even then, allowing different tabstop values would mess up alignments (like the end of `nvim-features-changed` above with the end of the `====` line).